From d700182849092e948366acb0b48dd10c61d9423b Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Sat, 21 Mar 2026 16:41:16 +0200
Subject: [PATCH] Remove the typec port controller

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 ...ra234-p3768-0000+p3767-xxxx-nv-common.dtsi | 28 -------------------
 1 file changed, 28 deletions(-)

diff --git a/nv-platform/tegra234-p3768-0000+p3767-xxxx-nv-common.dtsi b/nv-platform/tegra234-p3768-0000+p3767-xxxx-nv-common.dtsi
index 54338a0..a8cb94c 100644
--- a/nv-platform/tegra234-p3768-0000+p3767-xxxx-nv-common.dtsi
+++ b/nv-platform/tegra234-p3768-0000+p3767-xxxx-nv-common.dtsi
@@ -166,36 +166,8 @@
 			};
 		};
 
-		padctl@3520000 {
-			ports {
-				usb2-0 {
-					port {
-						typec_p0: endpoint {
-							remote-endpoint = <&fusb_p0>;
-						};
-					};
-				};
-			};
-		};
-
 		i2c@c240000 {
 			status = "okay";
-			fusb301@25 {
-				compatible = "onsemi,fusb301";
-				reg = <0x25>;
-				status = "okay";
-				#address-cells = <1>;
-				#size-cells = <0>;
-				interrupt-parent = <&gpio>;
-				interrupts = <TEGRA234_MAIN_GPIO(Z, 1) IRQ_TYPE_LEVEL_LOW>;
-				connector@0 {
-					port@0 {
-						fusb_p0: endpoint {
-							remote-endpoint = <&typec_p0>;
-						};
-					};
-				};
-			};
 		};
 
 		/* PWM1, 40pin header, pin 15 */
-- 
2.34.1

