From d583b7976809c39227d9660290ecd8e5de980528 Mon Sep 17 00:00:00 2001
From: Valentin Raevsky <valentin@compulab.co.il>
Date: Sun, 16 Nov 2025 13:21:14 +0200
Subject: [PATCH] Add EXTRAVERSION

Signed-off-by: Valentin Raevsky <valentin@compulab.co.il>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index a343118ba6a0..ff2d737f5543 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 VERSION = 5
 PATCHLEVEL = 15
 SUBLEVEL = 148
-EXTRAVERSION =
+EXTRAVERSION = -compulab
 NAME = Trick or Treat
 
 # *DOCUMENTATION*
-- 
2.34.1

